submissions/f3d0t/task_js_dom/script.js

```diff
@@ -0,0 +1,84 @@
+const pokemons = [
+	{
+		id: 121,
+		name: "Starmie",
+		description: "This Pokémon has an organ known as its core. The organ glows in seven colors when Starmie is unleashing its potent psychic powers.",
+		imagePath: "./img/121.png",
+	},
+	{
+		id: 201,
+		name: "Unown",
+		description: "This Pokémon is shaped like ancient writing. It is a mystery as to which came first, the ancient writings or the various Unown. Research into this topic is ongoing but nothing is known.",
+		imagePath: "./img/201.png",
+	},
+	{
+		id: 337,
+		name: "Lunatone",
+		description: "The phase of the moon apparently has some effect on its power. It’s active on the night of a full moon.",
+		imagePath: "./img/337.png",
+	},
+	{
+		id: 338,
+		name: "Solrock",
+		description: "When it rotates itself, it gives off light similar to the sun, thus blinding its foes.",
+		imagePath: "./img/338.png",
+	},
+	{
+		id: 790,
+		name: "Cosmoem",
+		description: "The king who ruled Alola in times of antiquity called it the “cocoon of the stars” and built an altar to worship it.",
+		imagePath: "./img/790.png",
+	},
+];
+
+const createMenuLinks = (array, container) => {
+	let wrapper = document.createElement("ul");
+	array.forEach((element) => {
+		let li = document.createElement("li");
+		li.textContent = element.name;
+		li.dataset.id = element.id;
+		li.className = "";
+		wrapper.appendChild(li);
+	});
+	container.appendChild(wrapper);
+};
+
+const paintContent = (id, arr) => {
+	let wrapper = document.createElement("main");
+	wrapper.className = "main";
+	let item = arr.find((item) => item.id == id);
```

Comment:
what you will do if find will return `undefined` ?

My suggestion: pass only item to this function. (If there is not such item don't call this function)

Reply:
@f3d0t not fixed